src/kernels/types.ts

```diff
@@ -740,7 +763,7 @@ export interface IKernelSocket {
     removeSendHook(hook: (data: any, cb?: (err?: Error) => void) => Promise<void>): void;
 }
 
-export type KernelSocketOptions = {
+export interface KernelSocketOptions {
```

Comment:
Changed to interface to improve finding references in VS Code via code lenses.
for some reason code lenses do not appear for members of types